bin/rails

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
@pmanrubia ok so `rake rails:update` doesn't touch that file at all. Railsdiff suggests the change: http://railsdiff.org/4.2.7.1/5.0.2 - should I re-instate the `__FILE__`? That might call into question the upgrade by following Railsdiff (except when this file was deleted, which I've put back and followed the Railsdiff change).